test/new-e2e/tests/agent-platform/install-script/install_script_test.go

```diff
@@ -186,3 +209,38 @@ func (is *installScriptSuite) DogstatsdAgentTest() {
 	common.CheckInstallationInstallScript(is.T(), client)
 	is.testUninstall(client, "datadog-dogstatsd")
 }
+
+type installScriptSuiteSysVInit struct {
+	e2e.BaseSuite[environments.Host]
+}
+
+func (is *installScriptSuiteSysVInit) TestInstallAgent() {
+	containerName := "installation-target"
+	host := is.Env().RemoteHost
+	client := common.NewDockerTestClient(host, containerName)
+
+	err := client.RunContainer("public.ecr.aws/ubuntu/ubuntu:22.04_stable")
```

Comment:
*question*: Should we use the 24.04 instead?

Reply:
Not sure, when I started I was using Debian, to be honest I'm not sure the exact version matters much as long as it's within the range of supported platforms.